src/net_processing.cpp

```diff
@@ -2522,10 +2521,16 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
         return;
     }
 
+    // BIP155 defines feature negotiation of addrv2 and sendaddrv2, which must happen
+    // between VERSION and VERACK.
     if (msg_type == NetMsgType::SENDADDRV2) {
         if (pfrom.fSuccessfullyConnected) {
-            // Disconnect peers that send SENDADDRV2 message after VERACK; this
-            // must be negotiated between VERSION and VERACK.
+            // Ignore peers that send a SENDADDRV2 message after VERACK.
+            return;
+        }
+        if (pfrom.m_wants_addrv2) {
+            // Disconnect peers that send SENDADDRV2 twice.
+            LogPrint(BCLog::NET, "peer=%d sent SENDADDRV2 twice; disconnecting\n", pfrom.GetId());
```

Comment:
This is not more harmful than an unknown message, so beside being dead code in practise, it is also not protecting against anything.

Reply:
dropped